native/packages/react-native-bpk-component-badge/README.md

```diff
@@ -44,8 +45,10 @@ export default class App extends Component {
 ```
 ## Props
 
-| Property              | PropType                                                                 | Required | Default Value |
+| Property              | PropType                                                                  | Required | Default Value |
 | --------------------- | --------------------------------------------------------------------------| -------- | ------------- |
 | message               | string                                                                    | true     | -             |
 | docked                | oneOf('start', 'end')                                                     | false    | null          |
+| iconSeparator         | string                                                                    | false    | null          |
```

Comment:
Let's make sure this is well documented. I was thinking we could have a heading underneath this table, something like:

```
## iconSeparator

`iconSeparator` is an optional string to place between icons when there are more than one, for example when you want to place a "+" between the icons.
```